src/components/progress-flow/progress-flow-item/progress-flow-item.scss

```diff
@@ -133,7 +133,7 @@
     z-index: z-index.$limel-progress-flow-divider;
     right: 0;
     transform: translateX(50%);
-    overflow-y: hidden;
+    overflow: hidden;
```

Comment:
It seems reasonable to me that the divider should have `overflow:hidden`, not just in the vertical direction, unless there's something I'm missing. The only thing that makes me the tiniest bit unconfident is, why was `overflow-y` used to begin with?

@Kiarokh You've been involved in this component before, right? Can you test this PR?

Reply:
Oh, yeah, checking this branch out is a bit special, since it's a branch from a separate fork. Don't worry about it Kia. I've tested it in the docs, and I feel good enough about merging that I'll just approve it myself.

Reply:
Sorry for the late response @adrianschmidt . It's first now I could open this notification. 

Honestly I don't remember whether we or I had a specific reason this. Maybe it was because of the alternative UI which we pulled out for now? 🤔 Not sure, even that doesn't really make sense. To me too, it sounds logical to have `overflow`.